migrations/900000000000001_readaccess_defaults.up.sql

```diff
@@ -0,0 +1,26 @@
+-- https://stackoverflow.com/questions/8092086/create-postgresql-role-user-if-it-doesnt-exist
+-- idempotent role creation
+DO
+$do$
+BEGIN
+IF EXISTS (
+  SELECT FROM pg_catalog.pg_roles
+  WHERE  rolname = 'readaccess') THEN
+  RAISE NOTICE 'Role "readaccess" already exists. Skipping.';
+ELSE
+  CREATE ROLE readaccess;
+END IF;
+END
+$do$;
```

Comment:
any reason this repeats https://github.com/mergestat/mergestat/blob/main/migrations/20220811202258_pg_read_only_user.up.sql ?